packages/graphql-playground-react/src/components/Playground/SchemaFetcher.ts

```diff
@@ -169,7 +159,9 @@ export class SchemaFetcher {
         },
         error: err => {
           reject(err)
+          log(err)
```

Comment:
Strip out all non-css changes please. Looks like a conflict merge/rebasing after the upgrade to Apollo's branch of playground.